modules/core/src/main/java/org/apache/ignite/internal/managers/communication/GridIoManager.java

```diff
@@ -1944,8 +1944,9 @@ public void sendToGridTopic(
         Message msg,
         byte plc
     ) throws IgniteCheckedException {
-        assert F.find(nodes, null, F.localNode(locNodeId)) == null :
-            "Internal Ignite code should never call the method with local node in a node list.";
+// TODO GG-21721 temporary allow this method on local node.
```

Comment:
@agoncharuk  I think we should fix GG-21721 before this PR will be merged to master!!!